setup.cfg

```diff
@@ -32,7 +32,7 @@ install_requires =
 	janus
 	ansible-runner
 	websockets
-	drools_jpy == 0.2.6
+	drools_jpy
```

Comment:
We need to pin the drools version. Why do you want to remove it?

Reply:
I need the next version of drools_jpy.  There is a fix there too.

Reply:
This PR can wait until that one is ready.

Reply:
In that case, this PR should not remove the drools_jpy version but bump it with the new version when it's published.